2009-10-18  Aurelien Jarno <aurelien@aurel32.net>

        * sysdeps/sparc/sparc32/sparcv9/memchr.S: remove, causes failures
	of test-memchr.

--- a/sysdeps/sparc/sparc32/sparcv9/memchr.S
+++ /dev/null
@@ -1,4 +0,0 @@
-#define ASI_PNF     0x82
-#define ASI_BLK_P   0xf0
-#define XCC icc
-#include <sparc64/memchr.S>
